src/handlers/comment/handlers/multiplier.ts

```diff
@@ -18,13 +18,26 @@ export const multiplier = async (body: string) => {
     return;
   }
 
-  const regex = /\/multiplier @(\w+) (\d+(?:\.\d+)?)/; // /multiplier @0xcodercrane 0.5
+  const regex = /(".*?"|[^"\s]+)(?=\s*|\s*$)/g;
 
   const matches = body.match(regex);
 
+  matches?.shift();
+
   if (matches) {
-    const username = matches[1];
-    const bountyMultiplier = parseFloat(matches[2]);
+    let bountyMultiplier = 1;
+    let username = "";
+    let reason = "";
+
+    for (const part of matches) {
+      if (!isNaN(parseFloat(part))) {
+        bountyMultiplier = parseFloat(part);
+      } else if (part.startsWith("@")) {
+        username = part.substring(1);
+      } else {
+        reason += part + " ";
```

Comment:
Why the additional space added here? 

```
reason += part + " ";
```

Reply:
I thought that the reason could be out of the double quote. So I added additional space. If no need, will update it.

Reply:
@rndquu fixed.